app/views/posts/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<h2> Edit Post</h2>
+
+<form action="/posts/<%= @post.id %>" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type="hidden" name="_method" value="put">
+
+  <label for="post[title]">Title:</label>
+  <input name="post[title]" value="<%= @post.title %>">
+
+  <label for="post[content]">content:</label>
+  <input name="post[content]" value="<%= @post.content %>">
+
+  <input type="submit" value="Edit">
+</form>
+
+<form action="/posts/<%= @post.id %>" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type="hidden" name="_method" value="delete">;
+  <input class="button-delete" type="submit" value="Delete!">
+</form>
```

Comment:
Highly recommend using Rails' built-in `form_for` helper. Will make writing out forms a lot easier.